adaptdl/adaptdl/speedup.py

```diff
@@ -72,14 +73,17 @@ def __init__(self, params, grad_params=None, init_batch_size=None,
                                     init_batch_size)
         else:
             self._max_batch_size = init_batch_size
+
+        self._gradient_accumulation = gradient_accumulation
```

Comment:
`self._accumulate = accumulate` is short and meaningful enough

Reply:
shortened the private var. I think the argument should still be the longer version to minimize ambiguity